### ISSUE TITLE
Device plugin 'Hermes Debugger (RN)' is not supported by the selected device 'iPhone 15 Pro' (iOS).

### ISSUE THREAD

**Issue by Hao-yiwen** [facebook/flipper#5432]
## 🐛 Bug Report
![截屏2024-01-21 16 25 27](https://github.com/facebook/flipper/assets/42827938/45ec56cd-70d6-477b-90f9-8cab6c31aa2e)

## To Reproduce
- why I can't use flipper to debugger my react-native project.
- Device plugin 'Hermes Debugger (RN)' is not supported by the selected device 'iPhone 15 Pro' (iOS).
- Plugin 'Hermes Debugger (RN)' is unavailable

## Environment
- macos: 14.2.1
- flipper: 0.239.0
- react-natvie: 0.73.2
- simulator: iphone 15pro (17.0)

**Comment by Hao-yiwen** [comment]
![截屏2024-01-21 16 29 04](https://github.com/facebook/flipper/assets/42827938/29c22d4f-8efd-4352-bf57-80e9f54520ed)

**Comment by krishnamudhiraj** [comment]
Hi @Hao-yiwen did you find the solution for this issue?

**Comment by Hao-yiwen** [comment]
No,I found many hermes engine all have this question, and I already start use flutter to develop app.
